src/nvim/log.c

```diff
@@ -284,13 +287,17 @@ static bool v_do_log_to_file(FILE *log_file, int log_level, const char *context,
                              va_list args)
   FUNC_ATTR_PRINTF(7, 0)
 {
+  // Name of the Nvim instance that produced the log.
+  // TODO(justinmk): introduce v:name?
+  static char name[16] = { 0 };
+
   static const char *log_levels[] = {
```

Comment:
I think it would be useful to define this list globally, so it can be re-used by the lua logger

https://github.com/neovim/neovim/blob/497046466e4a969cefce5fc2b7992dde157aa1a5/runtime/lua/vim/_editor.lua#L56-L65

and on that note, maybe it would then make sense to keep the name more verbose, e.g. `LOG_LEVEL_x`, to keep the two loggers more in sync.

Reply:
out of scope here, will be dealt with in https://github.com/neovim/neovim/pull/7062

the Lua logger is an accident. what do you mean "globally"? FFI is not an
option, we support puc Lua

Reply:
> out of scope here, will be dealt with in #7062

thanks for clarifying! 

> what do you mean "globally"? FFI is not an option, we support puc Lua

it seems like the idea is that `log.lua` should be entirely scrapped, in that case this doesn't really matter, but I'm curious, is it not possible to set `vim.log.levels` through `src/nvim/lua/stdlib.c`? sorry if it's a stupid question :)

Reply:
> , is it not possible to set `vim.log.levels` through `src/nvim/lua/stdlib.c`? sorry if it's a stupid question :)

could, but not worth the extra code. once we have a public interface, the log levels will be covered by tests, so the small amount of duplication is less costly than the extra code.